config/config.go

```diff
@@ -584,6 +667,14 @@ func New(v *viper.Viper) (*Configuration, error) {
 		c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap[bidderName] = struct{}{}
 	}
 
+	// To look for a special feature's vendor exceptions in O(1) time, we fill this hash table with bidders located in the
+	// VendorExceptions field of the GDPR.TCF2.SpecialFeature1 struct defined in this file
+	c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap = make(map[openrtb_ext.BidderName]struct{})
+	for v := 0; v < len(c.GDPR.TCF2.SpecialFeature1.VendorExceptions); v++ {
+		bidderName := c.GDPR.TCF2.SpecialFeature1.VendorExceptions[v]
+		c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap[bidderName] = struct{}{}
+	}
+
```

Comment:
This newly added block seems to simply be a repetition of the one above, comments and all
```
              665     // To look for a special feature's vendor exceptions in O(1) time, we fill this hash table with bidders located in the
              666     // VendorExceptions field of the GDPR.TCF2.SpecialFeature1 struct defined in this file
              667     c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap = make(map[openrtb_ext.BidderName]struct{})
              668     for v := 0; v < len(c.GDPR.TCF2.SpecialFeature1.VendorExceptions); v++ {
              669         bidderName := c.GDPR.TCF2.SpecialFeature1.VendorExceptions[v]
              670         c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap[bidderName] = struct{}{}
              671     }
              672
same as above 673     // To look for a special feature's vendor exceptions in O(1) time, we fill this hash table with bidders located in the
same as above 674     // VendorExceptions field of the GDPR.TCF2.SpecialFeature1 struct defined in this file
same as above 675     c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap = make(map[openrtb_ext.BidderName]struct{})
same as above 676     for v := 0; v < len(c.GDPR.TCF2.SpecialFeature1.VendorExceptions); v++ {
same as above 677         bidderName := c.GDPR.TCF2.SpecialFeature1.VendorExceptions[v]
same as above 678         c.GDPR.TCF2.SpecialFeature1.VendorExceptionMap[bidderName] = struct{}{}
same as above 679     }
              680
config/config.go
```

Reply:
Ah I think the duplicate code block here was introduced when merging with master. Removed.